yarn.lock

```diff
@@ -2896,13 +2896,13 @@ eslint-module-utils@^2.4.0:
     debug "^2.6.8"
     pkg-dir "^2.0.0"
 
-eslint-plugin-es@^1.4.0:
```

Comment:
please revent changes in `yarn.lock` as they are not related to the change you're making.

Reply:
fixed